terralab/commands/pipelines_commands.py

```diff
@@ -26,22 +26,27 @@ def list():
 
     for pipeline in pipelines_list:
         LOGGER.info(pipeline.pipeline_name)
+        LOGGER.info(indented("Version: " + str(pipeline.pipeline_version)))
         LOGGER.info(indented(pipeline.description))
 
 
 @pipelines.command(short_help="Get information about a pipeline")
 @click.argument("pipeline_name")
+@click.option("--version", type=int, help="pipeline version")
 @handle_api_exceptions
-def details(pipeline_name: str):
+def details(pipeline_name: str, version: int):
     """Get information about the PIPELINE_NAME pipeline"""
-    pipeline_info = pipelines_logic.get_pipeline_info(pipeline_name)
+    pipeline_info = pipelines_logic.get_pipeline_info(pipeline_name, version)
 
     # format the information nicely
     col_width = 16
```

Comment:
need to up this value to 18 it looks like to get this new output aligned properly